src/shared/utils/index.ts

```diff
@@ -127,7 +127,9 @@ export async function getLists(): Promise<Record<string, Set<string>>> {
       }),
     )
   })
-  await Promise.all(promises)
+  await Promise.all(
+    promises.map(async (promise) => promise.catch(async (err) => err)),
```

Comment:
The error should be logged.

Reply:
The error will still be logged as fetchValidatorList ran for the empty network.

Reply:
![Screenshot 2023-02-09 at 3 39 20 PM](https://user-images.githubusercontent.com/71317875/217933218-089ae88d-4d0c-4ee6-8e2b-fb59b5f92bb5.png)

Reply:
I will add to the error message to display network id

Reply:
If you catch and it returns this way the promise will resolve with the error.  Not a fan of that.  You should log it instead.  Additionally the async commands are not needed.

Reply:
Discussed with Caleb and added a comment to indicate that the error has been logged in `fetchValidatorList`

Reply:
Don't return the error.